can/io/asc.py

```diff
@@ -400,6 +405,10 @@ def log_event(self, message: str, timestamp: Optional[float] = None) -> None:
             formatted_date = time.strftime(
                 self.FORMAT_DATE.format(mlsec), time.localtime(self.last_timestamp)
             )
+            # changed 2022-08-04, moved file start header here to be one.
```

Comment:
Why?

Reply:
This appears to be a carry-over from when I `cherry-pick`ed from https://github.com/hardbyte/python-can/pull/1357/commits/ff7581bd20e5c8300a1d6ca64789067ac5387017. It looks like these lines need to be removed.